sample-linux.ini

```diff
@@ -5,10 +5,12 @@ portNumber = 8000
 ; If any symbols of interest aren't available locally (e.g. Windows DLLs), uncomment line below
 ; remoteSymbolServer = http://symbolapi.mozilla.org:80/
 
-; These options are unlikely to need editing by users
-maxCacheEntries = 2000000
-mruSymbolStateFile = /tmp/snappy-mru-symbols.json
-maxMRUSymbolsPersist = 10
+[MemoryCache]
+maxMemCacheFiles = 400
+
+[DiskCache]
+diskCachePath = /tmp/snappy/cache
+maxDiskCacheFiles = 1500
```

Comment:
Assuming most files are like xul.sym, and averege xul.sym pickled size is 15mb, this is about 22.5GB.
Looking at the centos machine, I see:

Filesystem      Size  Used Avail Use% Mounted on
/dev/xvda1      8.6G  2.8G  5.8G  33% /

The maxDiskCacheFiles will have to be smaller until there's more storage available. So 5.8GB / 15MB per file ~= 387. Let's set maxDiskCacheFiles to 300 files